src/test/java/com/hubspot/jinjava/lib/tag/ForTagTest.java

```diff
@@ -205,6 +205,27 @@ public void testForLoopWithDates() {
     assertEquals(new PyishDate(testDate).toString(), rendered);
   }
 
+  @Test
+  public void testTuplesWithPyList() {
+	  String template = "{% for href, caption in [('index.html', 'Index'), ('downloads.html', 'Downloads'), ('products.html', 'Products')] %}" + 
+	  		"<li><a href=\"{{href|e}}\">{{caption|e}}</a></li>\n" + 
+	  		"{% endfor %}";
+	  String expected = "<li><a href=\"index.html\">Index</a></li>\n" +
+	  		"<li><a href=\"downloads.html\">Downloads</a></li>\n" +
+	  		"<li><a href=\"products.html\">Products</a></li>\n";
+	  
+	  String rendered = jinjava.render(template, context);
```

Comment:
this could be broken out into a separate test

Reply:
done :+1: